crates/starfish/core/src/commit.rs

```diff
@@ -27,19 +27,12 @@ pub type CommitIndex = u32;
 
 pub(crate) const GENESIS_COMMIT_INDEX: CommitIndex = 0;
 
-/// Default wave length for all committers. A longer wave length increases the
-/// chance of committing the leader under asynchrony at the cost of latency in
-/// the common case.
-// TODO: merge DEFAULT_WAVE_LENGTH and MINIMUM_WAVE_LENGTH into a single
-// constant, because we are unlikely to change them via config in the
-// foreseeable future.
-pub(crate) const DEFAULT_WAVE_LENGTH: Round = MINIMUM_WAVE_LENGTH;
-
-/// We need at least one leader round, one voting round, and one decision round.
-pub(crate) const MINIMUM_WAVE_LENGTH: Round = 3;
+/// One wave consists of one leader round, one voting round, and one certifying
+/// round.
+pub(crate) const DEFAULT_WAVE_LENGTH: Round = 3;
```

Comment:
If it's hardcoded, then there is no need to prefix it with `DEFAULT_`. This will need to be properly replaced in other places as well. 

```suggestion
pub(crate) const WAVE_LENGTH: Round = 3;

```